eng/pipelines/runtime.yml

```diff
@@ -105,14 +105,20 @@ extends:
       - template: /eng/pipelines/common/platform-matrix.yml
         parameters:
           jobTemplate: /eng/pipelines/common/global-build-job.yml
-          buildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
+          buildConfig: Release
           platforms:
           - browser_wasm
           - browser_wasm_win
           jobParameters:
             nameSuffix: AllSubsets_CoreCLR
-            buildArgs: -s clr.runtime+libs -rc Release -c Release -lc $(_BuildConfig)
+            buildArgs: -s clr+libs -c $(_BuildConfig)
             timeoutInMinutes: 120
+            postBuildSteps:
+              - template: /eng/pipelines/common/templates/runtimes/build-runtime-tests.yml
+                parameters:
+                  testBuildArgs: test JIT/interpreter/Interpreter.csproj
```

Comment:
`<FeatureInterpreter>` is currently conditional on `('$(Platform)' == 'x64' OR '$(Platform)' == 'arm64') AND ('$(Configuration)' == 'debug' OR '$(Configuration)' == 'checked')` in `clr.featuredefines.props`. How will the interpreter tests pass in release configuration?

Reply:
goal of this PR is only to build it, not to make it run. It's indeed not working in Release yet.